README.md

```diff
@@ -544,67 +556,53 @@ into the subscription stream:
 Where `you-subscription-id` corresponds to the value of the `subscription` field
 which is returned with each websocket response.
 
-## ✅ Supported Features
+## 📚 Database Migration with Zero Downtime
 
 [⬅️ back to top](#-madara-starknet-client)
 
-### Starknet compliant
-
-Madara is compliant with the latest `v0.13.2` version of Starknet and `v0.7.1`
-JSON-RPC specs. You can find out more about this in the [interactions](#-interactions)
-section or at the official Starknet [JSON-RPC specs](https://github.com/starkware-libs/starknet-specs).
-
-### Feeder-Gateway State Synchronization
-
-Madara supports its own implementation of the Starknet feeder gateway, which
-allows nodes to synchronize state from each other at much faster speeds than
-a regular sync.
-
-> [!NOTE]
-> Starknet does not currently have a specification for its feeder-gateway
-> protocol, so despite our best efforts at output parity, you might still notice
-> some discrepancies between official feeder gateway endpoints and our own
-> implementation. Please let us know about if you encounter this by
-> [raising an issue](https://github.com/madara-alliance/madara/issues/new/choose)
-
-### State Commitment Computation
-
-Madara supports merkelized state verification through its own implementation of
-Besu Bonsai Merkle Tries. See the [bonsai lib](https://github.com/madara-alliance/bonsai-trie).
-You can read more about Starknet Block structure and how it affects state
-commitment [here](https://docs.starknet.io/architecture-and-concepts/network-architecture/block-structure/).
-
-### Database Migration
-
 When migration to a newer version of Madara you might need to update your
 database. Instead of re-synchronizing the entirety of your chain's state from
 genesis, you can use Madara's **warp update** feature.
 
-> [!NOTE]
-> Warp update requires an already synchronized _local_ node with a working
-> database.
+### Warp Update
+
+Warp update requires an already synchronized _local_ node with a working
+database. If you do not already have one, you can use the following command to
+generate a sample database:
+
+```bash
+cargo run --release --      \
```

Comment:
I think this is fine atm since this still remains the main way to get madara off and running. But I'm in agreement with you this will have to be updated once we have an actual cli people can install (I guess I could also add info on how to do that with `cargo install --git https://github.com/madara-alliance/madara --locked madara `, wdyt?)